### PR TITLE
Temaors/unit tests

### PR DESCRIPTION
**Pull request by temaors**

### CODE REVIEW

**Review by nikita-ermolaev-coral — APPROVED** [review]
good job

File: DIMS-Core/DIMS-Core.Tests/Repositories/TaskStateRepositoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        public async Task GetById_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+            // Act & Assert
+            await Assert.ThrowsAsync<InvArgException>(() => _fixture.Repository.GetById(id));
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/TaskStateRepositoryTests.cs]: Remove the space

File: DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            // Act
+            var deletedEntity = await _fixture.Context.TaskTracks.FindAsync(_fixture.EntityId);
+
+            // Assert
+            await Assert.ThrowsAsync<DbObjectIsNullException>(() => _fixture.Repository.GetById(_fixture.EntityId));
+            Assert.Null(deletedEntity);
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs]: remove space

File: DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        public async Task GetById_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+            // Act & Assert
+            await Assert.ThrowsAsync<InvArgException>(() => _fixture.Repository.GetById(id));
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs]: remove space

File: DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        public async Task Delete_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs:151]: add space before the method

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+        public async Task GetById_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+            // Act & Assert
+            await Assert.ThrowsAsync<InvArgException>(() => _fixture.Repository.GetById(id));
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs]: remove the space

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+            // Act
+            var deletedEntity = await _fixture.Context.UserProfiles.FindAsync(_fixture.EntityId);
+
+            // Assert
+            await Assert.ThrowsAsync<DbObjectIsNullException>(() => _fixture.Repository.GetById(_fixture.EntityId));
+            Assert.Null(deletedEntity);
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs]: remove the space

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+        public async Task Delete_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs:201]: add space before the method

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+        {
+            // Arrange
+            const int id = 0;
+
+            // Act & Assert
+            await Assert.ThrowsAsync<InvArgException>(() => _fixture.Repository.GetById(id));
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs]: remopve the space

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+            // Act
+            var deletedEntity = await _fixture.Context.UserTasks.FindAsync(_fixture.EntityId);
+            
+            // Assert
+            await Assert.ThrowsAsync<DbObjectIsNullException>(() => _fixture.Repository.GetById(_fixture.EntityId));
+            Assert.Null(deletedEntity);
+        }
+
+        [Fact]
+
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs]: remove the space

File: DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+        public async Task Delete_EmptyId_Fail()
+        {
+            // Arrange
+            const int id = 0;
+
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
Comment (dbogdan2000) [inline DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs:154]: add space before the method

**Review by dbogdan2000 — APPROVED** [review]

**Pull request merged**